tests/lib/testlib/portvlanlib.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+from libnmstate.schema import LinuxBridge
+
+
+def update_bridge_port_vlan_config(bridge, port_name, vlan_config):
```

Comment:
Currently this is not used, please either use it now or introduce it with the usage.

BTW, I prefer you make it immutable and return a new bridge state (do not bother if this is meant to just extract current code to a function).